lib/pages/signup/signup_page.dart

```diff
@@ -111,6 +112,14 @@ class _SignUpPageState extends State<SignUpPage> {
                           child: const Text("Sign Up"),
                         ),
 
+                        // Toggle button to login page
+                        TextButton(
+                          onPressed: () {
+                            Navigator.pushNamed(context, LoginPage.routeName);
```

Comment:
Dont use pushNamed
User can go back to the sign up page on back 
Fine for 1 or 2 times but maybe pushReplacement would be better

Reply:
Fixed [here](https://github.com/gdsc-nits-org/efficacy_admin/pull/20/commits/8b92d53731a18c935a0100e45cfb98f210c629b8)